pep-0592.rst

```diff
@@ -0,0 +1,295 @@
+PEP: 592
+Title: External annotations in the typing module
```

Comment:
Now I am thinking about a possible better title for this PEP. What do you think about "Flexible function and variable annotations"?

Reply:
"Mixing typing and non-typing annotations"? "Extra [meta]data in type hints"? "Non-typing information in type hints?" Just spitballing here.